apps/test/unit/templates/sectionProgress/VirtualizedSummaryViewTest.js

```diff
@@ -33,6 +33,7 @@ describe('VirtualizedSummaryView', () => {
         stages: [
           {
             id: 456,
+            position: 1,
```

Comment:
Is this already in the data we have, just not in tests?

Reply:
yes